app/views/thesis/deduplicate.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= content_for(:title, "Duplicate Theses Report | MIT Libraries") %>
+
+<% content_for :additional_js do %>
+  <script src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.min.js"></script>
+<% end %>
+
+<link href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.min.css" rel="stylesheet">
+
+<h3 class="title title-page">Theses with multiple authors</h3>
```

Comment:
`Theses with co-author` flag maybe? I don't think this actually shows the theses with multiple authors does it?

Reply:
No, as built this page shows only records with a populated "coauthor" field - not theses with multiple author records. That's part of what I suspect may come next, but not yet. I do think there's a reason to consider this title alongside the link in the nav, but that may be something we leave for QA testing.